### ISSUE TITLE
After second version bump everything starts to reanalyze

### ISSUE THREAD

**Issue by MartinNuc** [vscode-contrib/vscode-versionlens#66]
I used Version lens to update my dependencies one by one. After some update (I am on 1.7.2 right now) it starts to reanalyze all dependencies after bumping the second dependency.

All version informations disappear and I can see animating `Analyzing dependencies` icon in the right upper corner.

I recorded video for better understanding: https://youtu.be/rbdAfLnnDq8

**Comment by pflannery** [comment]
@MartinNuc Thanks for reporting this.

This looks related to #55

**Comment by pflannery** [comment]
@MartinNuc If you open the developer tools console. Do you see any errors?

![image](https://cloud.githubusercontent.com/assets/1727302/26037399/bf9562d8-38e9-11e7-90b7-1a5911267ff3.png)

**Comment by MartinNuc** [comment]
@pflannery There is no error. The only text I can see in dev console is json with new detected versions.

Example:

```
[Extension Host] {
  "@angular/platform-browser-dynamic": {
    "current": "4.1.2",
    "wanted": "4.0.2",
    "latest": "4.1.2",
    "location": "<path>"
  },
  "@angular/platform-server": {
    "current": "4.1.2",
    "wanted": "4.0.2",
    "latest": "4.1.2",
    "location": "<path>"
  }
}
```

**Comment by pflannery** [comment]
that's normal console output.

vscode is re-rendering code lenses unnecessarily which hopefully they will fix soon https://github.com/Microsoft/vscode/issues/26166

**Comment by pflannery** [comment]
@MartinNuc 

Mostly fixed in [v0.19.0](https://github.com/vscode-contrib/vscode-versionlens/blob/master/CHANGELOG.md#0190)

The only other issue I see now is where quickly replacing multiple code lenses is causing it to reload all code lenses. The discussion is still continuing at  Microsoft/vscode#26166

**Comment by MartinNuc** [comment]
Confirmed. It reloads all only when I do it quickly. When I wait for a visual version -> latest change it doesn't reload.

Thank you.
